hello_world_on_chain/test/user.ts

```diff
@@ -1,79 +1,92 @@
+// noinspection DuplicatedCode
+
 import hre from "hardhat";
 import {loadFixture} from "@nomicfoundation/hardhat-toolbox/network-helpers";
 import {expect} from "chai";
 import {ethers} from "ethers";
+import fs from "fs";
+import path from "path";
+import {User} from "../typechain-types";
 
-const fs = require("fs")
-const path = require("path")
+type EVMLinkData = {
+    publicKey: string;
+    transitiveAttestation: string;
+};
 
-function loadEVMLinkData(jsonPath: string) {
+function loadEVMLinkData(jsonPath: string): EVMLinkData {
     try {
-        const dir = path.resolve( __dirname, jsonPath);
-        const file = fs.readFileSync(dir);
+        const dir: string = path.resolve( __dirname, jsonPath);
+        const file: string = fs.readFileSync(dir, "utf8");
 
-        const data = JSON.parse(file);
+        const data: any = JSON.parse(file);
 
-        const k = data.enclave_attested_application_public_key.claims.public_key.data
-        const pubKeyBytes = ethers.decodeBase64(k)
-        const publicKeyHex = Buffer.from(pubKeyBytes).toString('hex');
+        const k: any =
+            data.enclave_attested_application_public_key.claims.public_key.data
+        const pubKeyBytes: Uint8Array = ethers.decodeBase64(k)
+        const publicKeyHex: string = Buffer.from(pubKeyBytes).toString('hex');
 
-        const taBytes = ethers.decodeBase64(data.transitive_attested_function_call.transitive_attestation)
-        const ta = Buffer.from(taBytes).toString('utf-8');
+        const taBytes:Uint8Array = ethers.decodeBase64(
+            data.transitive_attested_function_call.transitive_attestation
+        )
+        const ta: string = Buffer.from(taBytes).toString('utf-8');
 
-        return {
-            publicKey: `0x${publicKeyHex}`,
-            transitiveAttestation: ta
-        };
+        return { publicKey: `0x${publicKeyHex}`, transitiveAttestation: ta};
     } catch (e) {
-        console.log(`e`, e)
+        throw new Error(`Error loading EVM link data: ` + e);
```

Comment:
I've made all of these helper load functions throw an error, whereas previously they simply logged to the console and _implicitly_ returned an `undefined` value. This implicit value was causing issues further down in the test where we would get warnings about using the outputs of these functions because they were potentially undefined.

Now that all of their returns are explicit and they throw errors we do not get warnings.